ingenialink/utils/_utils.py

```diff
@@ -347,6 +347,7 @@ def set_logger_level(level):
 def convert_bytes_to_dtype(data, dtype):
     """Convert data in bytes to corresponding dtype."""
     __signed_dtypes_bytes = {REG_DTYPE.S8: 1, REG_DTYPE.S16: 2, REG_DTYPE.S32: 4}
+    __unsigned_dtypes_bytes = {REG_DTYPE.U8: 1, REG_DTYPE.U16: 2, REG_DTYPE.U32: 4}
```

Comment:
We could use [this ](https://github.com/ingeniamc/ingenialink-python/blob/aa737c5c5886770938c49ca31308375fd58f73e8/ingenialink/utils/_utils.py#L376) dict to simplify this function.

Reply:
Done!